content/reliability/docs/controlplane.md

```diff
@@ -147,6 +147,24 @@ users:
     token: <<super-admin sa’s secret>>
 ```
 
+## Admission Webhooks
+
+Kubernetes has two types of admission webhooks: [validating admission webhooks and mutating admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers). These allow a user to extend the kubernetes API and validate or mutate objects before they are accepted by the API. Poor configurations of these webhooks can distabilize the EKS control plane by blocking cluster critical operations.
+
+In order to avoid impacting cluster critical operations either avoid setting "catch-all" webhooks like the following:
+
+```
+- name: "pod-policy.example.com"
+  rules:
+  - apiGroups:   ["*"]
+    apiVersions: ["*"]
+    operations:  ["*"]
+    resources:   ["*"]
+    scope: "*"
+```
+
+Or make sure the webhook has a fail open policy with a timeout shorter than 30 seconds to ensure that if your webhook is unavailable it will not impair cluster critical workloads.
```

Comment:
Thanks for the contribution. This is good advice. Can you explain how you arrived at 30s?

Reply:
The default timeout for kube-api-server is 30 seconds. So if the webhook timeout is greater than 30 seconds it will fail because the kube-api-server timeout will kick in first. It's also recommended here: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts